docs/datamapper/index.rst

```diff
@@ -1,51 +1,205 @@
-============
-Data Mappers
-============
+===========
+DataMappers
+===========
 
-DataMappers, as the name suggest, map properties on themselves to your object.
+DataMappers function to reduce Python objects into simple data types supported 
+by JSON, and to expand JSON into Python objects. They are an alternative 
+approach to using Serialisers. 
 
-They allow you to easily write proxy objects, primarily for converting between
-serialised and live formats of your data.
+As the name suggests, a DataMapper will map properties on themselves to your 
+object. They allow you to easily write proxy objects, primarily for converting 
+between serialised (JSON) and live (Python) formats of your resources.
 
-DataMappers work using Python's descriptor protocol, which is most commonly
-used via the `property` built-in.
+.. Warning::
+    Since a DataMapper instance retain a reference to the object they are bound 
+    to, even when using << syntax, instances MUST NOT be shared between threads.
 
-When constructing a DataMapper you can pass an object for it to "bind" to. All
-field access to the DataMapper fields will proxy to this bound object.
+Field decorator: get/set
+========================
 
-Thread Safety
-=============
+You can set and get properties on a DataMapper by using Python's descriptor 
```

Comment:
reword...

    DataMappers leverage Python's "descriptor protocol" to allow you to control how properties are read from and written to your objects.